src/main/scala/treadle/TreadleTester.scala

```diff
@@ -74,8 +74,23 @@ class TreadleTester(annotationSeq: AnnotationSeq) {
 
   val startTime: Long = System.nanoTime()
 
+  private def findTopLevelClocks() = {
+    engine.symbolTable.symbols.collect {
+      case symbol if symbol.firrtlType == ClockType && !(symbol.name.contains(".") || symbol.name.endsWith("/prev")) =>
+        symbol
+    }.toList
+  }
+
   val clockInfoList: Seq[ClockInfo] = if (clockInfo.isEmpty) {
-    if (engine.symbolTable.contains("clock")) {
+    val topClocks = findTopLevelClocks()
+
+    if (topClocks.length > 2) {
+      println(s"Warning: multiple top level clocks found without any ClockInfo information, is this intentional?")
```

Comment:
Are you not using the logging infrastructure from `firrtl`?
Ideally you would use `logger.warn(...)` instead of `println(...)`

Reply:
Created issue #353 to deal with this